.gitpod.yml

```diff
@@ -1,4 +1,4 @@
-image: eosio/eosio-web-ide:v0.1.0
+image: eosio/eosio-web-ide:v0.1.1
```

Comment:
I made a similar comment on #4, should we be version locking these containers?  Remove the version?

Reply:
It reduces the risk of accidentally breaking people's environments. It allows them to choose when to upgrade to a newer container.